app/views/manage/questionnaires/_form.html.haml

```diff
@@ -50,11 +50,11 @@
             - if HackathonManager.field_enabled?(:why_attend)
               = f.input :why_attend, label: "Why #{HackathonConfig['name']}?", placeholder: "In a sentence or two, why would you like to attend #{HackathonConfig['name']}?", input_html: { rows: 3, maxlength: 280 }
 
-            = f.input :experience, as: :select, collection: Questionnaire::POSSIBLE_EXPERIENCES.invert, include_blank: "(select one...)", label: "Experience", input_html: { "data-validate" => "presence" }
-            = f.input :interest, as: :select, collection: Questionnaire::POSSIBLE_INTERESTS.invert, include_blank: "(select one...)", label: "Interest", input_html: { "data-validate" => "presence" }
```

Comment:
Are these being deleted because they're being moved to the model for validation? If so this should be reflected as part of the PR

Reply:
The logic is not being moved as the model already does this validation. data-validate: presence is somewhat of a mirror of that validation for the front end so the user gets instant feedback if they don't enter values in a required field.